assay_configs/HRD/eggd_conductor_dias_HRD_config_v1.0.0.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "Config for HRD assay",
+    "assay": "HRD",
+    "assay_code": "-9197|-10021",
+    "version": "1.0.0",
+    "details": "Includes 'demultiplexing and multiQC",
+    "changelog": {
+        "v1.0.0": "Initial working version"
+    },
+    "demultiplex": true,
+    "users": {
+        "org-emee_1": "CONTRIBUTE"
+    },
+    "executables": {
+        "applet-FvyXygj433GbKPPY0QY8ZKQG": {
+            "name": "multi_fastqc",
+            "details": "Takes all fastqs of all samples, starts one job which runs fastqc in sub jobs for each fastq",
+            "url": "https://github.com/eastgenomics/eggd_fastqc",
+            "analysis": "analysis_1",
+            "per_sample": false,
```

Comment:
just comparing to tso500, why is this set to false? Is this not per sample?

Reply:
TSO500 has been set as TRUE since multifastqc relies on the samplesheet, since there are PCAN samples in TSO runs, these need to be run seperately, example job(s): https://platform.dnanexus.com/panx/projects/GxQZKkQ4pK6GjZyJxP6YqZp6/monitor
Uranus runs only have 1 assay on 1 run which is why the multifastqc job can run on a per sample basis, example job: https://platform.dnanexus.com/panx/projects/GxG78Pj4Z72b8bv5Q7z03FZx/monitor/job/GxG7p304Z72ZY5XxyK9Y3Q6G

HRD runs will follow same convention as Uranus runs which is why it is set as false  
<!-- Reviewable comment -OEUKPlp47rS89eJ6k8r:b6wf5k9 -->
<!-- Sent from Reviewable.io -->